bazel/protobuf.bzl

```diff
@@ -153,19 +155,75 @@ def get_proto_arguments(protos, genfiles_dir_path):
     arguments = []
     for proto in protos:
         massaged_path = proto.path
-        if massaged_path.startswith(genfiles_dir_path):
+        if is_in_virtual_imports(proto):
+            incl_directory = get_include_directory(proto)
+            if massaged_path.startswith(incl_directory):
+                massaged_path = massaged_path[len(incl_directory) + 1:]
+        elif massaged_path.startswith(genfiles_dir_path):
             massaged_path = proto.path[len(genfiles_dir_path) + 1:]
         arguments.append(massaged_path)
+
     return arguments
 
 def declare_out_files(protos, context, generated_file_format):
     """Declares and returns the files to be generated."""
+
+    out_file_paths = []
+    for proto in protos:
+        if not is_in_virtual_imports(proto):
+            out_file_paths.append(proto.basename)
+        else:
+            path = proto.path[proto.path.index(_VIRTUAL_IMPORTS) + 1:]
+            # TODO: uncomment if '.' path is chosen over
```

Comment:
This is a effectively my question to you: Which one should we prefer (the uncommented path calculation, or the commented). Please check the PR description for more details:
---------------------------------
**Another option would be to skeep** the `_virtual_imports/<name_of_proto_library_target>` the suffix for the generated python stubs, which will make the path look like the following:
```
bazel-bin/google/firestore/v1beta1/google/cloud/firestore_v1beta1/proto/firestore_pb2.py
```

That will make using generated stubs simpler and cleaner (no need to specify imports argument), but it will make it inconsistent with the other rules (like cc_proto_library) and also more susseptible to naming conflicts (if there is already something under the generated path).
---------------------------------

To turn on the "other" option we would need to uncomment the TODO's. If we choose to keep the option 1 (i.e. the current state of this PR), I'll just delete the TODO's and the commented code segments they belong to.

Reply:
This is not needed anymore and was removed.